server/pulp/plugins/util/saxwriter.py

```diff
@@ -0,0 +1,94 @@
+from xml.sax.handler import ContentHandler
+from xml.sax.saxutils import escape, quoteattr
+
+
+class XMLWriter(ContentHandler):
+    """
+    XML writer similar to xml.sax.saxutils.XMLGenerator.
+    The result is indented XML, which is written sequentially to the stream.
+    """
+
+    def __init__(self, stream, encoding='utf-8', short_empty_elements=False):
```

Comment:
A doc block would be very helpful to make it clear what the new ``short_empty_elements`` argument does.

Generally, doc blocks for all of the methods in this class would be helpful.